src/alcotest/pprint.mli

```diff
@@ -0,0 +1,49 @@
+(*
+ * Copyright (c) 2013-2016 Thomas Gazagnaire <thomas@gazagnaire.org>
+ * Copyright (c) 2019      Craig Ferguson    <craig@tarides.com>
+ *
+ * Permission to use, copy, modify, and distribute this software for any
+ * purpose with or without fee is hereby granted, provided that the above
+ * copyright notice and this permission notice appear in all copies.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+ * WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+ * MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+ * ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+ * WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+ * ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+ * OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+ *)
+
+type path = [ `Path of string * int ]
+
+val pp_info : max_label:int -> doc_of_path:(path -> string) -> path Fmt.t
+
+type run_result =
+  [ `Ok
+  | `Exn of path * string * string
+  | `Error of path * string
+  | `Skip
+  | `Todo of string ]
+
+type event = [ `Result of path * run_result | `Start of path ]
+
+type result = {
+  success : int;
+  failures : int;
+  time : float;
+  errors : string list;
+}
+
+val pp_rresult_error : run_result Fmt.t
+
+val pp_event :
```

Comment:
`PPrint.pp_event` has a lot of `p`s :-) What about naming this module `Pp` and dropping the `pp_` prefix to have `Pp.event` ?

Reply:
Excellent idea; done.